### PR TITLE
TC-11

### PR DESCRIPTION
**Pull request by L4B0MB4**

### CODE REVIEW

File: backend/main.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+const port = 3000;
+const key = new NodeRSA({ b: 2064 });
+
+const text = "Hello RSA!";
+const encrypted = key.encrypt(text, "base64");
+console.log("encrypted: ", encrypted);
+const decrypted = key.decrypt(encrypted, "utf8");
+console.log("decrypted: ", decrypted);
+
+console.log(key.exportKey("public"));
```
Comment (CrVnsh) [inline backend/main.ts:15]: log

File: backend/main.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import * as NodeRSA from 'node-rsa';
+
+// the types are wrong for the express-validator
+const { query, validationResult } = require("express-validator");
+const port = 3000;
+const key = new NodeRSA({ b: 2064 });
+
+const text = "Hello RSA!";
+const encrypted = key.encrypt(text, "base64");
+console.log("encrypted: ", encrypted);
```
Comment (CrVnsh) [inline backend/main.ts:11]: log

File: backend/main.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+const decrypted = key.decrypt(encrypted, "utf8");
+console.log("decrypted: ", decrypted);
```
Comment (CrVnsh) [inline backend/main.ts:13]: log

**Review by CrVnsh — CHANGES_REQUESTED** [review]

File: package.json [hunk 4]
```diff
@@ -4,7 +4,8 @@
   "description": "## How to use in dev:",
   "scripts": {
     "start": "webpack --config webpack.config.js --watch --mode development",
-    "build": "webpack --config webpack.config.js --mode production"
+    "build": "webpack --config webpack.config.js --mode production",
+    "start-be": "webpack --config webpack.be.config.js --watch --mode development"
```
Comment (L4B0MB4) [inline package.json:8]: this will obviously change as soon as we have the monorepo

File: package.json [hunk 5]
```diff
@@ -18,13 +19,18 @@
...
     "ts-loader": "^8.0.3",
     "typescript": "^4.0.2",
     "webpack": "^4.44.1",
     "webpack-cli": "^3.3.12"
   },
   "dependencies": {
-    "@types/chrome": "^0.0.122"
+    "express": "^4.17.1",
+    "express-validator": "^6.6.1",
+    "node-rsa": "^1.1.1"
```
Comment (L4B0MB4) [inline package.json:34]: locked it up and its frontend compatible as well if you use webpack 👍

**Pull request closed without merging**